app/models/manageiq/providers/lenovo/manager_mixin.rb

```diff
@@ -74,7 +74,7 @@ def validate_connection(connection)
 
     def connection_rescue_block
       yield
-    rescue => err
+    rescue StandardError => err
```

Comment:
I may be in the minority but I don't think `rescue StandardError => err` is better than just `rescue => err`